### PR TITLE
Adding for WXKG01LM the possibility to enable a simple on-off mode

### PR DESCRIPTION
**Pull request by simonedimarzio**
Adding for WXKG01LM the possibility to enable a simple on-off mode:
- this option is useful when the device is connected to a common wall switch (when the click is never released in the <active> state)
- in this configuration only the 'single' and 'off' actions will alternate at each wall switch status change
- to enable this mode set 'mode_simple_on_off: true' in configuration.yaml device options

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -5204,36 +5204,51 @@ const converters = {
...
-                    publish({action: 'release', duration: duration});
-                    globalStore.putValue(msg.endpoint, 'hold', false);
+            if (options.mode_simple_on_off) {
+                // this option is useful when the device is connected to a common wall switch (when the click is never released in the <active> state)
+                // in this configuration only the 'single' and 'off' actions will alternate at each wall switch status change
+                // to enable this mode set 'mode_simple_on_off: true' in configuration.yaml device options
+                if (state === 0 || state === 1) {
+                    if (globalStore.getValue(msg.endpoint, 'click')) {
+                        globalStore.putValue(msg.endpoint, 'click', false);
+                        publish({action: 'single'});
```
Comment (Koenkk) [inline converters/fromZigbee.js:5214]: `action: 'on'` is better I think

**Comment by Koenkk** [comment]
I'm don't think this logic should be on the z2m level, the state is e.g. not persisted across restarts of z2m

**Comment by simonedimarzio** [comment]
> I'm don't think this logic should be on the z2m level, the state is e.g. not persisted across restarts of z2m

Any suggestions on where this logic shall be placed?
Anyway, I don’t think that loss of state after restart is a major issue, can be an assumption when using this mode.

Thanks
Simone

**Comment by Koenkk** [comment]
In your controller software (e.g. Home Assistant)

**Comment by simonedimarzio** [comment]
> In your controller software (e.g. Home Assistant)

Nope. The controller software does not receive any state change when the button is hold pressed.
The “simple mode” is done for this purpose, it triggers the controller software as soon as the switch state changes.

**Pull request closed without merging**